src/ripple/protocol/impl/tokens.cpp

```diff
@@ -30,13 +30,22 @@
 
 namespace ripple {
 
-static char rippleAlphabet[] =
+static constexpr char const* const alphabetForward =
```

Comment:
Not something I feel strongly about, but the `const` that follows `*` is redundant since `alphabetForward` is `constexpr`.  So, if you want, you can omit the second `const`.

Reply:
OK that looks good